cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -667,6 +667,15 @@ class HydratorPlusPlusTopPanelCtrl {
       });
   }
 
+  resetBtnsAndStopPoll() {
+    this.stopTimer();
+    this.updateTimerLabelAndTitle();
```

Comment:
could you please put some comments that explain this function `resetBtnsAndstopPoll`?